### PR TITLE
feat(volume3d): add orientation to volume viewport 3d

### PR DESCRIPTION
**Pull request by sedghi**
This will add `Orientation` to volume viewport 3D

### CODE REVIEW

File: packages/core/src/RenderingEngine/VolumeViewport3D.ts [hunk 1]
```diff
@@ -13,13 +13,27 @@ class VolumeViewport3D extends BaseVolumeViewport {
...
-    const { parallelProjection } = this.options;
+    const { parallelProjection, orientation } = this.options;
 
     const activeCamera = this.getVtkActiveCamera();
 
     if (parallelProjection != null) {
       activeCamera.setParallelProjection(parallelProjection);
     }
+
+    if (orientation && orientation !== OrientationAxis.ACQUISITION) {
```
Comment (jbocce) [inline packages/core/src/RenderingEngine/VolumeViewport3D.ts:24]: Actually... does it make sense to refactor this code and put it in the `BaseVolumeViewport` constructor? That is move it there and remove it from here and in `VolumeViewport`?

**Review by jbocce — COMMENTED** [review]
One comment that we really should consider.

File: packages/core/src/RenderingEngine/VolumeViewport3D.ts (hunk 1 above)
Comment (sedghi) [reply packages/core/src/RenderingEngine/VolumeViewport3D.ts:24]: sure

Comment (jbocce) [reply packages/core/src/RenderingEngine/VolumeViewport3D.ts:24]: Perhaps I was not clear... apologies. Why not put the whole `if` block (i.e. lines 37-40) in the `BaseVolumeViewport` `constructor`?

**Review by jbocce — COMMENTED** [review]
Apologies, I just wanted to clarify what I had meant.

File: packages/core/src/RenderingEngine/VolumeViewport3D.ts (hunk 1 above)
Comment (sedghi) [reply packages/core/src/RenderingEngine/VolumeViewport3D.ts:24]: because one has a early return but more logic later

**Review by jbocce — APPROVED** [review]
Looks good. Approved.

**Pull request merged**